ext/opcache/tests/jit/gh8461-007.phpt

```diff
@@ -0,0 +1,31 @@
+--TEST--
+Bug GH-8461 007 (JIT does not account for function re-compile)
+--EXTENSIONS--
+opcache
+--INI--
+opcache.enable=1
+opcache.enable_cli=1
+opcache.jit_buffer_size=1M
+opcache.jit=1255
+opcache.file_update_protection=0
+opcache.revalidate_freq=0
+opcache.protect_memory=1
+opcache.preload={PWD}/gh8461-007.inc
```

Comment:
You need to skip this test on Windows, since preloading is not supported there.

Reply:
thanks. done.